test/streamsearch.spec.js

```diff
@@ -221,7 +233,7 @@ describe('streamsearch', () => {
             expect(start).to.be.eql(expected[i][2]);
             expect(end).to.be.eql(expected[i][3]);
             i++;
-            if (i === 3) {
+            if (i >= 3) {
```

Comment:
is the change in behaviour acceptable here?

Reply:
Yes, if for some reason i get more chunks as expected I see the error that done was called multiple times. So I know I either expect not the right amount of events or I messed up the code.

Reply:
@Uzlopak Wouldn't it be better to add assert "i not larger than 3" instead then and not rely on mocha failing due to multiple done()?

Reply:
Not sure how mocha reacts to assertions failing after done was already invoked, though.

Reply:
It is by design that multiple done result in an error
https://github.com/mochajs/mocha/issues/216